internal/endian.go

```diff
@@ -1,3 +1,6 @@
+//go:build !arm64 && !amd64
```

Comment:
Can you use https://github.com/cilium/ebpf/blob/5fd10f532dbc2e31c5a3a1f715c4cef223998136/cmd/bpf2go/main.go#L44-L65 as a reference to make both little and big endian arches compile time?

Reply:
Done!